src/Analyzer.Utilities/Extensions/ISymbolExtensions.cs

```diff
@@ -66,6 +71,11 @@ public static bool IsUserDefinedOperator(this ISymbol symbol)
             return (symbol as IMethodSymbol)?.MethodKind == MethodKind.UserDefinedOperator;
         }
 
+        public static bool IsExplicitConversion(this ISymbol symbol)
```

Comment:
:bulb: Maybe this should be `IsConversionOperator`, since I believe it also returns true for `implicit operator T`.